hack/ci/e2e.sh

```diff
@@ -64,11 +64,20 @@ build() {
         /usr/local/bin/create_bazel_cache_rcs.sh || true
     fi
 
-    # build the node image w/ kubernetes
-    # TODO(bentheelder): remove the kube-root flag after we make kind try to
-    # find this in a go module compatible way
-    kind build node-image --type=bazel \
-        --kube-root="$(go env GOPATH)/src/k8s.io/kubernetes"
+    ARCH=$(go env GOARCH)
```

Comment:
we need to not fail if this fails, this script doesn't otherwise depend on go on the host.

Reply:
there's a script for this that the makefile uses

Reply:
My fault, I suggested it because I thought go was the preferred way.

I think this is the file https://github.com/kubernetes-sigs/kind/blob/master/hack/build/goarch.sh